README.md

```diff
@@ -113,7 +113,12 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
                 *    The selected URL will be checked to see if it is k-anonymous. If it is not, its k-anonymity will be incremented, but the `default URL` will be returned.
                 *    The reporting metadata will be used in the short-term to allow event-level reporting via `window.fence.reportEvent()` as described in the [FLEDGE explainer](https://github.com/WICG/turtledove/blob/main/Fenced_Frames_Ads_Reporting.md).
             *    There will be a per-origin (the origin of the Shared Storage worklet) budget for `selectURL`. This is to limit the rate of leakage of cross-site data learned from the selectURL to the destination pages that the resulting Fenced Frames navigate to. Each time a Fenced Frame navigates the top frame, for each `selectURL()` involved in the creation of the Fenced Frame, log(|`urls`|) bits will be deducted from the corresponding origin’s budget. At any point in time, the current budget remaining will be calculated as `max_budget - sum(deductions_from_last_24hr)`
-    *   Options can include `data`, an arbitrary serializable object passed to the worklet.
+    *   Options can include:
+        *   `data`, an arbitrary serializable object passed to the worklet. 
+        *   `retain` (defaults to false), a boolean denoting whether the worklet should be retained after it completes work for this call.
+            *   If `retain` is false or not specified, the worklet will shutdown as soon as the operation finishes and subsequent calls to it will fail.
```

Comment:
nit: prefer keep-alive as I think that's common parlance and more readily understood.

Reply:
OK, my concern with using "keep-alive" was that it could produce confusion with the "keep-alive" state that happens if the document is destroyed and there are still operations running.  

If you think that this ambiguity is not a problem, then I am happy to change it.

Reply:
I think this is higher priority in terms of keep-alive term usage, as it's part of the public API. We can rename the "keep-alive worklet" section to something like "Worklets can outlive renderers"

Reply:
OK, sounds good. Will do, thanks!

Reply:
Done.